docs/source/usingmame/defaultkeys.rst

```diff
@@ -1,106 +1,476 @@
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **J**                              | Player 1 Mahjong/Hanafuda J                                                                               |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **K**                              | Player 1 Mahjong/Hanafuda K                                                                               |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **L**                              | Player 1 Mahjong/Hanafuda L                                                                               |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **M**                              | Player 1 Mahjong/Hanafuda M                                                                               |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **N**                              | Player 1 Mahjong/Hanafuda N                                                                               |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **O**                              | Player 1 Mahjong/Hanafuda O                                                                               |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **P**                              | Player 1 Mahjong/Hanafuda P                                                                               |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **Q**                              | Player 1 Mahjong/Hanafuda Q                                                                               |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **Right Alt**                      | Player 1 Mahjong/Hanafuda Last Chance                                                                     |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **Right Ctrl**                     | Player 1 Mahjong/Hanafuda Score                                                                           |
```

Comment:
This should say “Take Score”, not just “Score”.  The source should be changed as well.

Reply:
Corrected in docs and code.